atuin/src/command/client/history.rs

```diff
@@ -266,6 +270,7 @@ impl Cmd {
         Ok(())
     }
 
+    #[allow(clippy::too_many_arguments)]
```

Comment:
@ellie @conradludgate Not happy about this one, but seems to be a decision of yours, to not use subcommand structs to pass arguments into methods.

Reply:
Not really a thought-out decision, It's just the easiest thing to do over time. This isn't a library so the chances of making mistakes with high arg count functions is low.

Reply:
Totally agree on this one. Having structs with many booleans is also not much better and will also break clippy. So I would consider this a reasonable decision. Even if it not a decision :)